package.json

```diff
@@ -51,6 +51,7 @@
     "connected-react-router": "^6.5.2",
     "dom-serializer": "^0.1.1",
     "install": "^0.13.0",
+    "kind-of": "^6.0.3",
```

Comment:
I don't think we use kind-off anywhere in the codebase. Did you use the command `npm run audit fix` to attempt to fix them?

Reply:
audit fix, did not fix nothing. When you open the package-lock.json, you'll find that kind-of is required as a dependency for multiple dependencies we use, like : is-data-descriptor, is-descriptor, is-number, merge-deep, micromatch, ... 
So we don't use it directly, but the modules we need, do

Reply:
@bdauria , as mentioned in the description, there's still (on the 19.10.x and 19.04.x branches) One high vulnerability reported by npm audit : 
![image](https://user-images.githubusercontent.com/34628915/86389083-19ab6280-bc96-11ea-8c10-1f0a0383de12.png)
This PR isn't intended to fix it. I let you two decide what to do and how to resolve it :